package.json

```diff
@@ -25,6 +25,7 @@
     "blakejs": "^1.2.1",
     "borsh": "^0.7.0",
     "curve25519-js": "^0.0.4",
+    "defillama-adapters": "^0.0.1-security.9",
```

Comment:
'This package contained malicious code and was removed from the registry by the npm security team. A placeholder was published to ensure users are not affected in the future.'